external-plugins/rehearse/plugin/main.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"k8s.io/client-go/rest"
+	v1 "k8s.io/test-infra/prow/client/clientset/versioned/typed/prowjobs/v1"
+	"net/http"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	"k8s.io/test-infra/prow/interrupts"
+
+	"os"
+	"time"
+
+	"k8s.io/test-infra/prow/git/v2"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/pkg/flagutil"
+	"k8s.io/test-infra/prow/config/secret"
+	prowflagutil "k8s.io/test-infra/prow/flagutil"
+	"kubevirt.io/project-infra/external-plugins/rehearse/events_server"
```

Comment:
Is good practice to re-organize imports separating groups by newline
- first golang
- second third parties grouped by project
- third app

Reply:
You are right. I let the IDE to organize this but I think that something was misconfigured there.